tests/e2e/filesForUpload/test-markdown.md

```diff
@@ -1,47 +1,35 @@
-# Presentation Viewer
----
-## Table of Contents
-1. Introduction
-   - ownCloud Web
-   - Presentation Viewer
-2. Usages 
-3. Why this App?
-4. How to use the extension? 
-5. Writing markdown? 
-6. Example of Markdown File
----
-## Introduction
---
-### ownCloud Web
-A new view on your file platform
---
-### Presentation Viewer
-Extension for ownCloud Web
----
-## Usages
-Allows users to create slide presentations directly from markdown files
----
-## Why this App?
-- To reduce your time for formatting
-- You will get more time on content
-- Uniform design for organization presentations
+# My Markdown File
+
+## Heading 2
+
+### Heading 3
+
+#### Heading 4
+
+## [reveal.js](https://revealjs.com) in oCIS
+
 ---
-## How to use the extension?
-- Right-click on the resource
-- Click _Open in Presentation Viewer_
+
+# Ordered Lists
+1. First item in an 'ordered' list
```

Comment:
I know this is just demo content, but is there any reason why you use '' here and not in the following items?